### PR TITLE
Fix SendBox styling

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Fixed SendBox styling to align with what is on Figma.
Figma:
image
Screenshots from storybook after fix:
- normal
image
- focused
image
- has input
image
- has system message
image
- has system message in dark theme
image
- disabled
image

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2425344

# How Tested

Tested different SendBox states in storybook and compared with Figma

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/SendBox.tsx [hunk 1]
```diff
@@ -155,7 +146,15 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
             }
             onTyping && onTyping();
           }}
-          styles={concatStyleSets(TextFieldStyleProps, { fieldGroup: styles?.textField })}
+          styles={concatStyleSets(
```
Comment (JamesBurnside) [inline packages/react-components/src/components/SendBox.tsx:149]: pull out the style concat function into a const as per https://github.com/microsoft/fluentui/wiki/Performance#good-1

File: packages/react-components/src/components/SendBox.tsx [hunk 2]
```diff
@@ -177,11 +176,22 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
           {onRenderIcon ? (
             onRenderIcon(props, isMouseOverSendIcon)
           ) : (
-            <SendIcon className={mergeStyles(sendIconDiv, styles?.sendMessageIcon)} outline={!isMouseOverSendIcon} />
+            <SendIcon
+              className={mergeStyles(
```
Comment (JamesBurnside) [inline packages/react-components/src/components/SendBox.tsx:180]: same here

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/components/SendBox.tsx [hunk 3]
```diff
@@ -131,10 +121,11 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   const textTooLongMessage = textValueOverflow ? TEXT_EXCEEDS_LIMIT : undefined;
+  const errorMessage = systemMessage ?? textTooLongMessage;
```
Comment (allenplusplus) [inline packages/react-components/src/components/SendBox.tsx:124]: Should we rename the props systemMessage to errorMessage?

**Review by allenplusplus — APPROVED** [review]

File: packages/react-components/src/components/SendBox.tsx (hunk 3 above)
Comment (mgamis-msft) [reply packages/react-components/src/components/SendBox.tsx:124]: Hmmm MessageThread also uses systemMessage though and I think we should keep it consistent. Should we also change MessageThread's systemMessage to errorMessage? I would be down to do that just want to know others' thoughts.

File: packages/react-components/src/components/SendBox.tsx (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-components/src/components/SendBox.tsx]: Good point. Thanks.

**Pull request merged**